pkg/sql/logictest/testdata/logic_test/zone_config_system_tenant

```diff
@@ -104,7 +104,6 @@ SELECT crdb_internal.pb_to_json('cockroach.roachpb.SpanConfig', config)
 FROM system.span_configurations
 WHERE end_key > (SELECT crdb_internal.table_span($t_id)[1])
 ----
-{"gcPolicy": {"ttlSeconds": 90000}, "numReplicas": 3, "rangeMaxBytes": "16777216", "rangeMinBytes": "1048576"}
```

Comment:
is this deletion expected?

Reply:
Got deleted due to a --rewrite race with the logictest `retry` directive (just waits for 2s and spits out query results). I added it back, and edited the TTL seconds by hand, and it runs successfully without --rewrite. We need something better than just `retry` for --rewrite in these logic tests.